src/waypoint.jsx

```diff
@@ -333,13 +352,18 @@ export default class Waypoint extends React.Component {
    * @return {Object}
    */
   render() {
+    if (this.props.children) {
+      return <div ref={this.refElement}>{this.props.children}</div>;
```

Comment:
Unfortunately, it turned out the refToMeasure approach doesn't work, because the `ref` of the child is `null` until after the parent has rendered.

On the other hand, cloning and injecting a ref does work after all. I was afraid of overwriting a `ref`, but it was easy to preserve.